scripts/auto-rebase/rebase.sh

```diff
@@ -901,18 +1057,34 @@ usage() {
 command=${1:-help}
 case "$command" in
     to)
-        [[ $# -ne 3 ]] && usage
-        rebase_to "$2" "$3"
+        [[ $# -ne 4 ]] && usage
+        rebase_to "$2" "$3" "$4" "$5"
         ;;
     download)
-        [[ $# -ne 3 ]] && usage
+        [ $# -ne 4 ] && usage
         download_release "$2" "$3"
+        download_lvms_operator_bundle "$4" "$5"
         ;;
     changelog) update_changelog;;
     buildfiles) update_buildfiles;;
     go.mod) update_go_mod;;
     generated-apis) regenerate_openapi;;
-    images) update_images;;
-    manifests) update_manifests;;
+    images)
+        update_images
+        update_lvms_images
+        ;;
+    manifests)
+        update_manifests
+        update_lvms_manifests
+        ;;
+    csi)
```

Comment:
This may have been a debugging option, but would we ever want to update just csi images? We do not have special option for ovn, etc., so why csi is special?

Reply:
We're free to remove this.  It was initially for debugging, but b/c LVMS isn't tethered to OCP z-streams (i.e. isn't tracked by the ocp-release image), I figured it'd be useful to have should we need to pull in OCP y-stream compatible versions ad hoc.

Reply:
I think I'd rather remove this option to keep it similar with the other components.